packages/react-hooks/src/inbox/use-inbox-actions.ts

```diff
@@ -102,9 +99,19 @@ const useInboxActions = (): IInboxActions => {
       });
     };
 
+  const handleMarkAsRead = async (messageId: string, fromWS?: boolean) => {
+    dispatch(markMessageRead(messageId));
+    if (!fromWS) {
+      await inboxClient.markRead(messageId);
+    }
+  };
+
   const handleInit: IInboxActions["init"] = async (payload) => {
     dispatch(initInbox(payload));
-    handleGetUnreadMessageCount();
+    const dateDiff = getDateDiff(payload.lastMessagesFetched);
+    if (!dateDiff || dateDiff > 3600000) {
```

Comment:
only refetch unread count on reload if its been an hour

Reply:
Could `dateDiff` be zero? If so, `!dateDiff` would eval to `true`. Which, based on the `dateDiff > 3600000` might be undesirable.

Might be safer to do:
```suggestion
    if (dateDiff === undefined || dateDiff > 3600000) {
```